.github/workflows/pr-validation.yml

```diff
@@ -0,0 +1,34 @@
+name: "PR Tasks Completed Check"
+on:
+  pull_request:
+    types: opened
+
+jobs:
+  task-check:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - name: Comment if checkmark is missing
+        if: ${{ !contains(github.event.pull_request.body, '[X] I acknowledge') }}
+        uses: actions/github-script@v6
+        with:
+          script: |
+            github.rest.issues.createComment({
+            issue_number: context.issue.number,
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            body: "Please don't submit a PR containing Kata solutions. If you are adding an improvement to the Katas repo, please resubmit this PR and check the `[X]` box in the PR template."
```

Comment:
Perhaps change this line to "Please don't submit a Pull Request containing a Kata solution to the original repo from Emily Bache. If you are instead trying to add an improvement, please resubmit this PR and check the ...